docs/data/data-grid/list-view/list-view.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Data Grid - List view
+---
+
+# Data Grid - List view [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+<p class="description">Display data in a single-column list view. Can be used to present a more compact grid on smaller screens and mobile devices.</p>
+
+List view can be enabled by providing the `unstable_listView` prop.
+
+Unlike the default grid view, the list view makes no assumptions on how data is presented to end users.
+
+In order to display data in a list view, a `unstable_listColumn` prop must be provided with a `renderCell` function.
+
```

Comment:
Perhaps we could add a small note on the `unstable_` prefix, considering that not only developers will likely check this feature out.
Something like:
``````suggestion
:::info
Features with APIs prefixed by `unstable_` are ready for production; we use this prefix to indicate that the API might change based on early feedback.
:::
``````

Reply:
Good idea, added a note here https://github.com/mui/mui-x/pull/14393/commits/2a2734d674d2a4693950a932c90ebd43a5077f3b